### ISSUE TITLE
New pricingplanmanager events are unknown to the linter

### ISSUE THREAD

**Issue by jim-barber-he** [aws-cloudformation/cfn-lint#4303]
### CloudFormation Lint Version

cfn-lint 1.41.0

### What operating system are you using?

Debian Linux

### Describe the bug

When I view distributions in the AWS CloudFront console it is triggering AccessDenied errors for the `pricingplanmanager:ListSubscriptions` action.
I guess it has something to do with this announcement: https://aws.amazon.com/about-aws/whats-new/2025/11/aws-flat-rate-pricing-plans/

I added CloudFormation to allow this IAM action to our IAM Identity center permissions, but `cfn-lint` errors like so:
```
W3037 'pricingplanmanager' is not one of ['a2c', 'a4b', 'access-analyzer', 'account', 'acm', 'acm-pca', 'action-recommendations', 'activate', 'aiops', 'airflow', 'airflow-serverless', 'amplify', 'amplifybackend', 'amplifyuibuilder', 'aoss', 'apigateway', 'app-integrations', 'appconfig', 'appfabric', 'appflow', 'application-autoscaling', 'application-signals', 'application-transformation', 'applicationinsights', 'appmesh', 'appmesh-preview', 'apprunner', 'appstream', 'appstudio', 'appsync', 'apptest', 'aps', 'arc-region-switch', 'arc-zonal-shift', 'arsenal', 'artifact', 'athena', 'auditmanager', 'autoscaling', 'autoscaling-plans', 'aws-marketplace', 'aws-marketplace-management', 'aws-portal', 'awsconnector', 'b2bi', 'backup', 'backup-gateway', 'backup-search', 'backup-storage', 'batch', 'bcm-dashboards', 'bcm-data-exports', 'bcm-pricing-calculator', 'bcm-recommended-actions', 'bedrock', 'bedrock-agentcore', 'billing', 'billingconductor', 'braket', 'budgets', 'bugbust', 'cases', 'cassandra', 'ce', 'chatbot', 'chime', 'cleanrooms', 'cleanrooms-ml', 'cloud9', 'clouddirectory', 'cloudformation', 'cloudfront', 'cloudfront-keyvaluestore', 'cloudhsm', 'cloudsearch', 'cloudshell', 'cloudtrail', 'cloudtrail-data', 'cloudwatch', 'codeartifact', 'codebuild', 'codecatalyst', 'codecommit', 'codeconnections', 'codedeploy', 'codedeploy-commands-secure', 'codeguru', 'codeguru-profiler', 'codeguru-reviewer', 'codeguru-security', 'codepipeline', 'codestar', 'codestar-connections', 'codestar-notifications', 'codewhisperer', 'cognito-identity', 'cognito-idp', 'cognito-sync', 'comprehend', 'comprehendmedical', 'compute-optimizer', 'config', 'connect', 'connect-campaigns', 'consoleapp', 'consolidatedbilling', 'controlcatalog', 'controltower', 'cost-optimization-hub', 'cur', 'customer-verification', 'databrew', 'dataexchange', 'datapipeline', 'datasync', 'datazone', 'dax', 'dbqms', 'deadline', 'deepcomposer', 'deepracer', 'detective', 'devicefarm', 'devops-guru', 'directconnect', 'discovery', 'dlm', 'dms', 'docdb-elastic', 'drs', 'ds', 'ds-data', 'dsql', 'dynamodb', 'ebs', 'ec2', 'ec2-instance-connect', 'ec2messages', 'ecr', 'ecr-public', 'ecs', 'eks', 'eks-auth', 'elasticache', 'elasticbeanstalk', 'elasticfilesystem', 'elasticloadbalancing', 'elasticmapreduce', 'elastictranscoder', 'elemental-activations', 'elemental-appliances-software', 'elemental-support-cases', 'elemental-support-content', 'emr-containers', 'emr-serverless', 'entityresolution', 'es', 'events', 'evidently', 'evs', 'execute-api', 'finspace', 'finspace-api', 'firehose', 'fis', 'fms', 'forecast', 'frauddetector', 'freertos', 'freetier', 'fsx', 'gamelift', 'gameliftstreams', 'geo', 'geo-maps', 'geo-places', 'geo-routes', 'glacier', 'globalaccelerator', 'glue', 'grafana', 'greengrass', 'groundstation', 'groundtruthlabeling', 'guardduty', 'health', 'healthlake', 'honeycode', 'iam', 'identity-sync', 'identitystore', 'identitystore-auth', 'imagebuilder', 'importexport', 'inspector', 'inspector-scan', 'inspector2', 'internetmonitor', 'invoicing', 'iot', 'iot-device-tester', 'iotanalytics', 'iotdeviceadvisor', 'iotevents', 'iotfleethub', 'iotfleetwise', 'iotjobsdata', 'iotmanagedintegrations', 'iotsitewise', 'iottwinmaker', 'iotwireless', 'iq', 'iq-permission', 'ivs', 'ivschat', 'kafka', 'kafka-cluster', 'kafkaconnect', 'kendra', 'kendra-ranking', 'kinesis', 'kinesisanalytics', 'kinesisvideo', 'kms', 'lakeformation', 'lambda', 'launchwizard', 'lex', 'license-manager', 'license-manager-linux-subscriptions', 'license-manager-user-subscriptions', 'lightsail', 'logs', 'lookoutequipment', 'lookoutmetrics', 'lookoutvision', 'm2', 'machinelearning', 'macie2', 'managedblockchain', 'managedblockchain-query', 'mapcredits', 'marketplacecommerceanalytics', 'mechanicalturk', 'mediaconnect', 'mediaconvert', 'mediaimport', 'medialive', 'mediapackage', 'mediapackage-vod', 'mediapackagev2', 'mediastore', 'mediatailor', 'medical-imaging', 'memorydb', 'mgh', 'mgn', 'migrationhub-orchestrator', 'migrationhub-strategy', 'mobileanalytics', 'mobiletargeting', 'monitron', 'mpa', 'mq', 'neptune-db', 'neptune-graph', 'network-firewall', 'network-security-director', 'networkflowmonitor', 'networkmanager', 'networkmanager-chat', 'networkmonitor', 'nimble', 'notifications', 'notifications-contacts', 'oam', 'observabilityadmin', 'odb', 'omics', 'one', 'opensearch', 'opsworks', 'opsworks-cm', 'organizations', 'osis', 'outposts', 'panorama', 'partnercentral', 'partnercentral-account-management', 'payment-cryptography', 'payments', 'pca-connector-ad', 'pca-connector-scep', 'pcs', 'personalize', 'pi', 'pipes', 'polly', 'pricing', 'private-networks', 'profile', 'proton', 'purchase-orders', 'q', 'qapps', 'qbusiness', 'qdeveloper', 'qldb', 'quicksight', 'ram', 'rbin', 'rds', 'rds-data', 'rds-db', 'redshift', 'redshift-data', 'redshift-serverless', 'refactor-spaces', 'rekognition', 'repostspace', 'resiliencehub', 'resource-explorer', 'resource-explorer-2', 'resource-groups', 'rhelkb', 'robomaker', 'rolesanywhere', 'route53', 'route53-recovery-cluster', 'route53-recovery-control-config', 'route53-recovery-readiness', 'route53domains', 'route53profiles', 'route53resolver', 'rtbfabric', 'rum', 's3', 's3-object-lambda', 's3-outposts', 's3express', 's3tables', 's3vectors', 'sagemaker', 'sagemaker-data-science-assistant', 'sagemaker-geospatial', 'sagemaker-mlflow', 'sagemaker-unified-studio-mcp', 'savingsplans', 'scheduler', 'schemas', 'scn', 'sdb', 'secretsmanager', 'security-ir', 'securityhub', 'securitylake', 'serverlessrepo', 'servicecatalog', 'servicediscovery', 'serviceextract', 'servicequotas', 'ses', 'shield', 'signer', 'signin', 'simspaceweaver', 'sms', 'sms-voice', 'snow-device-management', 'snowball', 'sns', 'social-messaging', 'sqlworkbench', 'sqs', 'ssm', 'ssm-contacts', 'ssm-guiconnect', 'ssm-incidents', 'ssm-quicksetup', 'ssm-sap', 'ssmmessages', 'sso', 'sso-directory', 'sso-oauth', 'states', 'storagegateway', 'sts', 'support', 'support-console', 'supportapp', 'supportplans', 'sustainability', 'swf', 'synthetics', 'tag', 'tax', 'textract', 'thinclient', 'timestream', 'timestream-influxdb', 'tiros', 'tnb', 'transcribe', 'transfer', 'transform', 'translate', 'trustedadvisor', 'ts', 'user-subscriptions', 'uxc', 'vendor-insights', 'verified-access', 'verifiedpermissions', 'voiceid', 'vpc-lattice', 'vpc-lattice-svcs', 'vpce', 'waf', 'waf-regional', 'wafv2', 'wam', 'wellarchitected', 'wickr', 'wisdom', 'workdocs', 'worklink', 'workmail', 'workmailmessageflow', 'workspaces', 'workspaces-instances', 'workspaces-web', 'xray']
```

### Expected behavior

The new `pricingplanmanager` IAM actions to be known by cfn-lint.

### Reproduction template

I doubt there's a need to supply a whole template. The inline policy I was trying to add to the permission set contained this new entry:
```yaml
            {
              "Action": "pricingplanmanager:ListSubscriptions",
              "Effect": "Allow",
              "Resource": "*"
            },
```

**Comment by kddejong** [comment]
Should be covered by #4295
